--- a/net/minecraft/util/EncoderCache.java
+++ b/net/minecraft/util/EncoderCache.java
@@ -36,1 +36,1 @@
-            return EncoderCache.this.cache.getUnchecked(new EncoderCache.Key<>(codec, input, ops)).map(value -> value instanceof Tag tag ? tag.copy() : value);
+            return EncoderCache.this.cache.getUnchecked(new EncoderCache.Key<>(codec, input, ops)).map(value -> (T)(value instanceof Tag tag ? tag.copy() : value));
