--- a/net/minecraft/util/datafix/fixes/OptionsKeyLwjgl3Fix.java
+++ b/net/minecraft/util/datafix/fixes/OptionsKeyLwjgl3Fix.java
@@ -9,0 +9,1 @@
+import com.mojang.serialization.Dynamic;
@@ -139,1 +140,1 @@
-         input -> input.update(DSL.remainderFinder(), tag -> tag.getMapValues().map(map1 -> tag.createMap(map1.entrySet().stream().map(entry -> {
+         input -> input.update(DSL.remainderFinder(), tag -> tag.getMapValues().<Dynamic<?>>map(map1 -> tag.createMap(map1.entrySet().stream().map(entry -> {
